frame/contracts/src/migration.rs

```diff
@@ -400,6 +404,29 @@ mod v9 {
 	}
 }
 
+mod v10 {
+	use super::*;
+	use crate::storage::DeletedContract;
+
+	#[storage_alias]
+	type DeletionQueue<T: Config> = StorageValue<Pallet<T>, Vec<DeletedContract>>;
+
+	pub fn migrate<T: Config>(weight: &mut Weight) {
+		let Some(contracts) = DeletionQueue::<T>::take() else { return };
+		weight.saturating_accrue(T::DbWeight::get().reads_writes(1, 1));
+
+		let mut queue = crate::DeletionQueue::<T>::load();
```

Comment:
I should not use that and only manipulate Struct / storage structures that are defined within this v10 migration module right?